docker-compose.deploy.yml

```diff
@@ -697,6 +697,7 @@ services:
       - CERT_PUBLIC_KEY_PATH=/run/secrets/jwt-public-key.{{ts}}
       - MONGO_URL=mongodb://metrics:${METRICS_MONGODB_PASSWORD}@mongo1/metrics?replicaSet=rs0
       - HEARTH_MONGO_URL=mongodb://hearth:${HEARTH_MONGODB_PASSWORD}@mongo1/hearth-dev?replicaSet=rs0
+      - DASHBOARD_MONGO_URL=mongodb://performance:${HEARTH_MONGODB_PASSWORD}@mongo1/hearth-dev?replicaSet=rs0
```

Comment:
you are selecting the wrong db here. instead of "hearth-dev" it should be "performance". And the password should be "PERFORMANCE_MONGODB_PASSWORD"